source/includes/Visit/visits.md

```diff
@@ -5,12 +5,11 @@
 * A Visit in OpenMRS represents precisely what it sounds like: a time when a patient is actively interacting with the 
 the healthcare system, typically at a location.
 
-* The metadata differentiating different types of visits is a Visit Type. Visit Types displayed in the user interface, 
-also, can be searched against.
+* The metadata differentiating different types of visits is a Visit Type. Visit Types displayed in the user interface, also can be searched against.
```

Comment:
@Akayeshmantha  what does this line actually mean I dint get it completely ??

Reply:
Visit type is metadata which differentiates different types of visits.